packages/kotti-ui/source/kotti-popover/KtPopover.vue

```diff
@@ -19,13 +26,27 @@
 import { createPopper } from '@popperjs/core'
 import { mixin as clickaway } from 'vue-clickaway'
 
+import KtPopoverIconTextItem from './KtPopoverIconTextItem.vue'
+
+const optionIsValid = (option) =>
+	(!option.isDisabled || typeof option.isDisabled === 'boolean') &&
```

Comment:
what if you get `option: { isDisabled: '' , ... }` 
do you want to allow this?
you want to be more strict because `!` will negate any primitive (so empty string will pass, number zero will pass

so what do we actually want? you either want to not get `isDisabled` prop at all on your `option` object or you want to get it as a boolean, you don't want API mis-use and sources of bugs.
so your check should be as strict as `isDisabled` is undefined or `isDisabled` has a `boolean` type

just a note: 
```typeof undefined === 'undefined``` so you can run the typeof check and see if it's either this or that